model-optimizer/mo/utils/class_registration.py

```diff
@@ -128,7 +128,7 @@ def dump_graph_for_graphviz(self, node_attrs: list = [], edge_attrs: list = [],
         if nodes_to_dump is None:
             nodes_to_dump = self.nodes()
         string = '\ndigraph {\n'
-        string += 'node [color=lightblue2, style=filled];\n'
+        string += 'node [color=lightblue2, style=filled shape="box"];\n'
```

Comment:
Is comma missed?

Reply:
The code works and without comma. But I add it to avoid confusion.